lib/rspec/core/configuration.rb

```diff
@@ -1389,8 +1389,11 @@ def self.delegate_to_ordering_manager(*methods)
 
       # @macro delegate_to_ordering_manager
       #
-      # Sets the default global order and, if order is `'rand:<seed>'`, also
-      # sets the seed.
+      # Sets the default global ordering strategy. By default this can be one
+      # of `:defined`, `:random`, but is customisable through the
```

Comment:
We've tend to use the American spelling with a `z`:

```
➜  repos git:(master) ack customize | wc -l
      65
➜  repos git:(master) ack customise | wc -l
       2
```

Mind spelling it with a `z` for consistency?